packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -48,6 +48,12 @@ Future<int> runTests(
   Directory coverageDirectory,
   bool web = false,
 }) async {
+  // Configure package:test to use the Flutter engine for child processes.
+  final String shellPath = artifacts.getArtifactPath(Artifact.flutterTester);
+  if (!processManager.canRun(shellPath)) {
+    throwToolExit('Cannot find Flutter shell at $shellPath');
```

Comment:
Maybe something like "Cannot execute tester at $shellPath"

Reply:
Done. I actually copied this text from below, in the same file :)